invenio_vocabularies/resources/resource.py

```diff
@@ -52,7 +52,7 @@ class VocabulariesResourceConfig(RecordResourceConfig):
         "type": ma.fields.Str(required=True),
     }
 
-    request_args = VocabularySearchRequestArgsSchema
+    request_search_args = VocabularySearchRequestArgsSchema
```

Comment:
This got fixed in master this morning 👼 ... sorry, hope the rebase is not too tricky

Reply:
That was bound to happen with PRs sitting there a while... Rebase is no issue.